shared/src/commonMain/kotlin/org/hyperskill/app/leaderboard/domain/model/LeaderboardItem.kt

```diff
@@ -0,0 +1,15 @@
+package org.hyperskill.app.leaderboard.domain.model
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import org.hyperskill.app.users.domain.model.User
+
+@Serializable
+data class LeaderboardItem(
+    @SerialName("user")
+    val user: User,
```

Comment:
Here we can't use `Profile` model, backend uses different serializers for `/api/profiles` and `/api/users`